### PR TITLE
Port the "point" interactive graph type (with fixed number of points) to Mafs

### PR DESCRIPTION
**Pull request by benchristel**
Issue: https://khanacademy.atlassian.net/browse/LEMS-1776

## Test plan:

Run this command to get data for point graphs with fixed numbers of points
onto your clipboard:

```
grep -rl '"type":"point"' data/questions/ | xargs cat | grep -v unlimited | pbcopy
```

Then paste the data into the flipbook (`yarn dev`,
`http://localhost:5173/#flipbook`).

You should be able to move the points and answer questions successfully.

There is a known bug with keyboard interaction on some of the graphs:
https://khanacademy.atlassian.net/browse/LEMS-1885

Co-authored-by: Mark Fitzgerald <markfitzgerald@khanacademy.org>

### CODE REVIEW

File: packages/perseus/src/widgets/interactive-graphs/reducer/interactive-graph-reducer.test.ts [hunk 1]
```diff
@@ -31,7 +46,7 @@ describe("moveControlPoint", () => {
             moveControlPoint(0, [5, 6], 0),
         );
 
-        expect(updated.coords[0]).toEqual([
+        expect(updated.coords?.[0]).toEqual([
```
Comment (mark-fitzgerald) [inline packages/perseus/src/widgets/interactive-graphs/reducer/interactive-graph-reducer.test.ts:49]: Didn't PR1132 make `coords` required?

Comment (benchristel) [reply packages/perseus/src/widgets/interactive-graphs/reducer/interactive-graph-reducer.test.ts]: Fixed!

**Review by mark-fitzgerald — APPROVED** [review]
Looks good (but I'm biased since I am the other half of the pair on this work).

**Comment by benchristel** [comment]
I don't understand why codecov says `getDefaultPoints` is not covered. It definitely is.

**Pull request merged**